zh_CN

```diff
@@ -0,0 +1 @@
+add chinese for pbx
```

Comment:
List was manually downloaded and pushed as ef9e04a424bb3c8a36c8d78bb5fa85e35d784b3a